pygmt/src/image.py

```diff
@@ -49,6 +50,14 @@ def image(self, imagefile, **kwargs):
         [**+p**\ [*pen*]][**+r**\ [*radius*]][**+s**\ [[*dx*/*dy*/][*shade*]]].
         If set to ``True``, draw a rectangular border around the image
         using :gmt-term:`MAP_FRAME_PEN`.
+    bitcolor : str or list
```

Comment:
Is a list allowed?

Reply:
A list can be used to set both foreground and background colors of 1-bit images:
```python
import pygmt

fig = pygmt.Figure()

for i_G in [
        None,
        "blue+b",
        ["blue+b", "red+f"],
]:
    fig.image(imagefile="@vader1.png", G=i_G)  
    fig.shift_origin(xshift="1c")

fig.show()
# fig.savefig(fname="image_Q_list.png")
```
_Output figure_:
![image_Q_list](https://github.com/GenericMappingTools/pygmt/assets/94163266/1341fc1f-ea3e-4418-8144-a25125a0c415)

Reply:
Ah I see, thanks for the example! So the list would translate to `-Gblue+b -Gred+f`. Makes sense.

Reply:
Exactly! Maybe moving the sentence "Can be repeated with different settings." upwards makes this a bit clearer (please see commit https://github.com/GenericMappingTools/pygmt/pull/2615/commits/951a11072cef0a72bda75b2f4f1ca76dac3eb967)?